docs/wsgi.rst

```diff
@@ -1,193 +0,0 @@
-.. _wsgi-chapter:
-
-=============================
-Running Kitsune with mod_wsgi
-=============================
```

Comment:
Here we can mention the alternative of running sumo with gunicorn instead of the django dev server.

Reply:
It sounds like there's actually some snippets in here we might want to keep? I'm not sure I have the brain space to keep+rewrite this page until Friday, so I'm tempted to un-delete and slap the "this may be outdated" warning on it.